manifests/charts/cloudcore/templates/rbac_cloudcore_feature.yaml

```diff
@@ -6,13 +6,13 @@ metadata:
   labels:
     rbac/aggregate-to-cloudcore: "true"
 rules:
-- apiGroups: [""]
-  resources: ["serviceaccounts"]
-  verbs: ["list", "watch"]
-- apiGroups: ["policy.kubeedge.io"]
-  resources: ["serviceaccountaccesses"]
-  verbs: ["list", "watch"]
-- apiGroups: ["rbac.authorization.k8s.io"]
-  resources: ["roles", "clusterrolebindings", "rolebindings", "clusterroles"]
-  verbs: ["list", "watch"]
+  - apiGroups: [ "" ]
```

Comment:
It's enough to add space between array items. No need to add space at the beginning or end of each line or array.

Reply:
ok